docs(treeview):OnDoubleClick and OnContextMenu events (#499)
telerik/blazor-docs: components/treeview/events.md

}
````

## OnItemDoubleClick

> The `OnItemDoubleClick` event fires as a response to the user double clicking on a TreeView node.

Reviewer: Simpler

```suggestion
The `OnItemDoubleClick` event fires when the user double-clicks a TreeView node.
```

On a side note, we may mention what happens on touch devices. Does this event ever fire?

Author: Indeed, checking with Stamo if the behavior will be the same as in the Grid. Will update the doc tomorrow morning.

Author: Confirmed, OnItemDoubleClick fires on double tap for mobile devices. Updated.

        items.Add(new TreeItem()
        {
            Id = 4,
            Text = "site.psd",
            ParentId = 2,
            HasChildren = false,
            Icon = "psd",
            Expanded = true
        });
        items.Add(new TreeItem()
        {
            Id = 5,
            Text = "index.js",
            ParentId = 3,
            HasChildren = false,
            Icon = "js"
        });
        items.Add(new TreeItem()
        {
            Id = 6,
            Text = "index.html",
            ParentId = 3,
            HasChildren = false,
            Icon = "html"
        });
        items.Add(new TreeItem()
        {
            Id = 7,
            Text = "styles.css",
            ParentId = 3,
            HasChildren = false,
            Icon = "css"
        });

        FlatData = items;
    }
    #endregion

    #region Data Model

    public class TreeItem
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public int? ParentId { get; set; }
        public bool HasChildren { get; set; }
        public string Icon { get; set; }
        public bool Expanded { get; set; }
    }
    #endregion
}
````

### OnItemContextMenu

> The `OnItemContextMenu` event fires as a response to the user right clicking on a TreeView node, the context menu keyboard button or long-touch for mobile devices. 

Reviewer: ```suggestion
The `OnItemContextMenu` event fires when the user right-clicks on a TreeView node, presses the context menu keyboard button or taps and holds on a mobile device. 
```
